web_client/components/containers/analysis.jsx

```diff
@@ -119,6 +119,10 @@ const AnalysisContainer = connect(
               ) {
                 result = false;
               }
+            } else if (item._modelType === 'folder') {
+              if (item.name === '__osumo_tmp') {
```

Comment:
I wonder if this "magic string" should be put in a constants file and imported as a symbol.